spec/export/index.md

```diff
@@ -230,7 +236,8 @@ Note: When requesting status, the client SHOULD use an ```Accept``` header for i
       "error" : [{
         "type" : "OperationOutcome",
         "url" : "http://serverpath2/err_file_1.ndjson"
-      }]
+      }],
+      "extension":{"<a href="http://myserver.example.org/extra-property">http://myserver.example.org/extra-property</a>": true}
```

Comment:
There shouldn't be any HTML / href in this example extension

Reply:
Yup i fixed that. I do have a question though. Is this supposed to be formated like a normal FHIR extension? 

Currently based on the comment it looks like this (below) which is unlike the normal array with the url, valueX extension model. Im just making sure the content below is what's intended.
  ```json
    {
      "transactionTime": "[instant]",
      "request" : "[base]/Patient/$export?_type=Patient,Observation",
      "requiresAccessToken" : true,
      "output" : [{
        "type" : "Patient",
        "url" : "http://serverpath2/patient_file_1.ndjson"
      },{
        "type" : "Patient",
        "url" : "http://serverpath2/patient_file_2.ndjson"
      },{
        "type" : "Observation",
        "url" : "http://serverpath2/observation_file_1.ndjson"
      }],
      "error" : [{
        "type" : "OperationOutcome",
        "url" : "http://serverpath2/err_file_1.ndjson"
      }]
      }],
      "extension":{"http://myserver.example.org/extra-property": true}
    }
  ```

Reply:
> Is this supposed to be formated like a normal FHIR extension?

No, since this response body is not a FHIR resource, but rather an ad-hoc JSON structure, we've decided to adopt the same extension conventions as [CDS Hooks](https://cds-hooks.org).